.claude/commands/settings-integration.md

```
@@ -0,0 +1,365 @@
+    'id'             => 'tag_line',           // CRITICAL: must match WP REST args key for nested objects
+    'type'           => 'field',
+    'variant'        => 'text',               // text | show_hide | color_picker | switch | wp_media_upload
+    'label'          => __('Tag Line', 'my-plugin'),
+    'section_id'     => 'app_identity',       // groups this field under its section
+    'value'          => $settings['tag_line'] ?? '',
+    'description'    => __('App tagline text.', 'my-plugin'),
+]
+```
+
+**Available variants:**
+| Variant | Use for |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)